tests/installation/system_role/select_role_transactional_server.pm

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Select System Role (Transactional Server) and navigate to next screen
+#          using REST API.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+
+sub run {
+    my $system_role = $testapi::distri->get_system_role_controller();
+    $system_role->select_system_role('transactional_server');
+    $system_role->next();
```

Comment:
let's apply here the 'next' approach to hide the next from the test itself:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13291